actions/invite_actions.jsx

```diff
@@ -37,9 +37,10 @@ export function sendMembersInvites(teamId, users, emails) {
             } catch (e) {
                 response = {error: localizeMessage('invite.members.unable-to-add-the-user-to-the-team', 'Unable to add the user to the team.')};
             }
+            const errors = response.data ? (response.data.errors || {}) : {};
```

Comment:
Show we be handling for `response.error` object as well? What happens if all users were invalid?

Reply:
@sudheerDev the new method addTeamMembersGracefully will return all errors inside the 'response.data.errors', not in the error field. So we are ignoring it

Reply:
@reflog , probably i am missing something here. From the looks of it https://github.com/mattermost/mattermost-redux/pull/1010/files#diff-69b6b6dec7aa6243100f46236dad4f7eR472 returns an error object. So i am assuming there are still error only cases without `data.errors`.I thought if all users fail we return only error. Correct me if i am wrong.  

And if all errors are handled with `response.data.errors` i think we can remove the try catch above to make it a little explicit.

Reply:
Yes, all errors are handled with `response.data.errors` (see https://github.com/mattermost/mattermost-server/pull/13372/files#diff-d6e8e8eda818a9467cff95c90993b15fR594). I'll remove the try catch to make this clearer

Reply:
Thanks got it, I was curious about error handling of this API as it seemed new to me. If that is the case you probably don't even need to check for `response.data`? It is a blocking one question though.

Reply:
Never mind i guess we need the check for API auth issues i guess